src/public/sass/ITCSS/06_components/pages/error-404/_error-404.scss

```
@@ -0,0 +1,49 @@
+@use '../../../01_settings/colors' as *;
+@use '../../../01_settings/global' as *;
+@use '../../../01_settings/fonts' as *;
+
+.error-404 {
+  display: flex;
+  max-width: 50rem;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 0 1rem;
+  margin: 3rem auto;
+  color: $color-gray;
+  font-weight: $weight-light;
+  text-align: center;
+
+  &__image {
+    width: 60%;
+    aspect-ratio: 4/3;
+  }
+
+  &__heading {
+    margin-top: 0;
+    margin-bottom: 0;
+    color: $color-black;
+    font-size: $font-size-h1;
+  }
+
+  &__message {
+    margin-top: 1rem;
+  }
+
+  &__btn {
+    padding: 0.375rem 0.625rem;
+    border: none;
+    border-radius: 0.3125rem;
+    margin-top: 1.5rem;
```

Maybe you can reduce this value
```suggestion
    margin-top: 0.5rem;
```